src/op_analytics/transforms/fees/update/04_agg_daily_trace_to.sql

```diff
@@ -0,0 +1,257 @@
+    , cast(count_internal_calls_static_type_trace_success, 'UInt32') AS preagg_count_internal_calls_static_type_trace_success
+    , cast(count_internal_calls_delegate_type, 'UInt32') AS preagg_count_internal_calls_delegate_type
+    , cast(count_internal_calls_delegate_type_trace_success, 'UInt32') AS preagg_count_internal_calls_delegate_type_trace_success
+    , cast(count_internal_calls_call_type, 'UInt32') AS preagg_count_internal_calls_call_type
+    , cast(count_internal_calls_call_type_trace_success, 'UInt32') AS preagg_count_internal_calls_call_type_trace_success
+
+    , cast(count_traces_type_all_types, 'UInt32') AS preagg_count_traces_type_all_types
+    , cast(count_traces_trace_type_call, 'UInt32') AS preagg_count_traces_trace_type_call
+    , cast(count_traces_trace_type_suicide, 'UInt32') AS preagg_count_traces_trace_type_suicide
+    , cast(count_traces_trace_type_create, 'UInt32') AS preagg_count_traces_trace_type_create
+    , cast(count_traces_trace_type_create2, 'UInt32') AS preagg_count_traces_trace_type_create2
+    , cast(count_traces_trace_type_create_any, 'UInt32') AS preagg_count_traces_trace_type_create_any
+
+    , cast(is_transaction_with_internal_call_type_call, 'Bool') AS is_transaction_with_internal_call_type_call
+    , cast(is_transaction_with_internal_call_type_call_or_delegate, 'Bool') AS is_transaction_with_internal_call_type_call_or_delegate
+
+  FROM
+    blockbatch_gcs.read_date(
+      rootpath = 'blockbatch/aggregated_traces/agg_traces_tr_to_hash_v1'
+      , chain = '*'
+      , dt = { dtparam: Date }
+    )
+  WHERE
+    NOT is_system_transaction
+)
+
+SELECT
+  dt
+  , chain
+  , chain_id
+  , network
+  , trace_to_address
+
+  -- Aggregate transaction metrics, in transactions when a given contract was called
+  /*
+    Note: We can NOT sum these up at the project level, since a project may have
+    many of its contracts called in a single transaction.
+
+    For these use cases (i.e. project impact metrics), we can aggregate `aggregated_traces_tr_to_hash`
+    at the project name level, only after all trace_to_address entries are mapped to their respective
+    project name (i.e. a transaction may call 10 contracts, but this may only represent 3 projects).
+  */
+  -- Attributes
+  , COUNT(*) AS count_transactions_called
+  , countIf(tx_success) AS count_transactions_called_tx_success
+
+  , COUNT(DISTINCT block_number) AS count_distinct_blocks_called
+  , COUNT(DISTINCT tx_from_address) AS count_distinct_tx_from_addresses
+  , COUNT(DISTINCT tx_to_address) AS count_distinct_tx_to_addresses
+  , COUNT(DISTINCT tx_method_id) AS count_distinct_tx_method_ids
+
+  -- Transaction-Level gas & fees
+  , SUM(tx_l2_gas_used) AS sum_tx_l2_gas_used
+  , sumIf(tx_l2_gas_used, tx_success) AS sum_tx_l2_gas_used_tx_success
+
+  , SUM(tx_fee_native) AS sum_tx_fee_native
+  , sumIf(tx_fee_native, tx_success) AS sum_tx_fee_native_tx_success
+
```

Comment:
we can cut all `tx_success` variants from here on down: `sum_tx_l1_fee_native_tx_success` - `sum_tx_l2_legacy_extra_fee_native_tx_success`